tests/source/single-line-simple-if.rs

```diff
@@ -0,0 +1,109 @@
+// rustfmt-single_line_simple_if: true
+// rustfmt-unstable_features: true
```

Comment:
The `// rustfmt-unstable_features: true` comment shouldn't be necessary when writing tests.